packages/node/src/setup-config.ts

```diff
@@ -42,8 +42,12 @@ export async function setupConfig(cwd: string, opts: SetupOptions = {}) {
   } else {
     const content = outdent`
 import { defineConfig } from "@pandacss/dev"
+import presetBase from "@pandacss/preset-base"
+import presetPanda from "@pandacss/preset-panda"
 
 export default defineConfig({
+    // The set of reusable and shareable configuration presets.
+    presets: [presetBase, presetPanda],
```

Comment:
Let's use the string version of the preset so users don't have to install these packages.

`presets: ['@pandacss/preset-base', '@pandacss/preset-panda'],`